src/js/routes/services.js

```diff
@@ -1,37 +1,69 @@
 import Router from 'react-router';
-let Route = Router.Route;
+let {Route, Redirect} = Router;
 
+import DeploymentsTab from '../pages/services/DeploymentsTab';
 import ServiceOverlay from '../components/ServiceOverlay';
 import ServicesPage from '../pages/ServicesPage';
+import ServicesTab from '../pages/services/ServicesTab';
 
-let servicesRoutes = {
+let services = {
```

Comment:
That's an interesting point. Happy to obey the convention, but since we import it using `import services from 'services.js';` should we think about updating the module names too?